src/index.js

```diff
@@ -0,0 +1,154 @@
+import './style.css';
+import { addTask, todoItems } from './todo.js';
+import updateCompletedStatus from './status.js';
+import select2 from '../images/select.png';
+import delete2 from '../images/delete.png';
+import refresh from '../images/refresh.png';
+import edit from '../images/edit.jpg';
+
+const container = document.getElementById('container');
+const form = document.querySelector('form');
+const ul = document.querySelector('.todoList');
+form.innerHTML = `
+<label class="title"
+>Today's To Do <img class="refresh" src=${refresh} alt="refresh"
+/></label>
+<input
+class="inputText"
+placeholder="Add to your list..."
+type="text"
+/>
+`;
+
+const btnDeleteAll = document.createElement('button');
+btnDeleteAll.setAttribute('class', 'delete-all');
+btnDeleteAll.textContent = 'Clear all completed';
+container.append(btnDeleteAll);
+
+const display = () => {
+  todoItems.sort((a, b) => a.index - b.index);
+  ul.innerHTML = ''; // clear the list before re-rendering it
+  for (let i = 0; i < todoItems.length; i += 1) {
+    const node = document.createElement('li');
+    node.setAttribute('class', 'todo-item');
+    node.setAttribute('data-key', todoItems[i].index);
+    node.setAttribute('class', 'todo-item editable');
+
+    node.innerHTML = ` 
+        <input class="checkbox" id="${todoItems[i].index}" type="checkbox"/>
+        <label for="${todoItems[i].index}" class="option">
+            <img class="select" src=${select2} alt=""/>
+            <img class="delete" src=${delete2} alt=""/>
+            <img class="edit" src=${edit} alt="edittext"/>
+        </label>
+        <input class="items" type="text" value="${todoItems[i].text}" readonly />
+
+    `;
+    ul.append(node);
+
+    // Get the selectdots, deleteoption, and editoption elements for this task
+    const selectdots = node.querySelector('.select');
+    const deleteoption = node.querySelector('.delete');
+    const editoption = node.querySelector('.edit');
+    // Get the checkbox element for this task
```

Comment:
**JS Best practices**
- Cont naming: kindly consider to change these const to camelCase format 🐫

Reply:
Suggested changes have been incorporated within the project. Screen shot is attached below for your kind reference.
![Uploading image.png…]()

Reply:
Suggested changes have been incorporated within the project. Screen shot is attached below for your kind reference.
![image](https://user-images.githubusercontent.com/117091923/223144193-5fe79557-4d58-478a-b3f5-955024e7ee17.png)